<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>741</width>
    <height>593</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>741</width>
    <height>593</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>741</width>
    <height>593</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>乳腺超声肿瘤分割系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon/icon.png</normaloff>icon/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>721</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainFrame">
     <item>
      <layout class="QVBoxLayout" name="fileinfo">
       <item>
        <layout class="QHBoxLayout" name="fileNameLayout">
         <item>
          <widget class="QLabel" name="fileNameLabel">
           <property name="text">
            <string>文件名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fileNameShow">
           <property name="text">
            <string>File Name</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="uploadDateLayout">
         <item>
          <widget class="QLabel" name="uploadDateLabel">
           <property name="text">
            <string>日  期：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateTimeEdit" name="dateTimeEdit"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="timeCountLayout">
         <item>
          <widget class="QLabel" name="timeCountLabel">
           <property name="text">
            <string>耗  时：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="timeCountShow">
           <property name="text">
            <string>   秒</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="segline">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="imginfo">
       <item>
        <layout class="QVBoxLayout" name="srcimginfo">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QGraphicsView" name="srcimgshow"/>
           </item>
           <item>
            <widget class="QPushButton" name="uploadButton">
             <property name="text">
              <string>选择文件</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="resultinfo">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QGraphicsView" name="resultimgshow"/>
           </item>
           <item>
            <widget class="QPushButton" name="saveButton">
             <property name="text">
              <string>保存结果</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="saveInfoLayout">
       <item>
        <widget class="QLabel" name="saveStateLabel">
         <property name="text">
          <string>保存状态：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="savePathLabel">
         <property name="text">
          <string>结果未保存</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
